src/java/com/palantir/cassandra/metrics/FailureDetectorMetrics.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.cassandra.metrics;
+
+import java.net.InetAddress;
+
+import com.google.common.net.InetAddresses;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Histogram;
+import org.apache.cassandra.gms.FailureDetector.ArrivalWindow;
+import org.apache.cassandra.metrics.CassandraMetricsRegistry;
+
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
+
+public class FailureDetectorMetrics
+{
+    public static void register(InetAddress ep, ArrivalWindow window)
+    {
+        Metrics.register(createMetricName(ep, "phi"), (Gauge<Double>) window::getLastReportedPhi);
+        Metrics.register(createMetricName(ep, "interval_histogram"), new Histogram(window));
+        Metrics.register(createMetricName(ep, "last_interval"), (Gauge<Long>) window::getLastInterval);
```

Comment:
Nit changes: considering how they appear in grafana we probably wanna rename this to sth like PhiLastInterval 
We should also match the CamelCase format of other metrics for consistency